src/main/java/uk/gov/companieshouse/officerfiling/api/controller/OfficerFilingControllerImpl.java

```diff
@@ -97,15 +98,18 @@ public ResponseEntity<Object> createFiling(@PathVariable final String transId,
         final var passthroughHeader =
                     request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader());
         final var transaction = transactionService.getTransaction(transId, passthroughHeader);
-        logger.infoContext(transId, "transaction found", logMap);
+        logger.infoContext(transId, "transaction found", new LogHelper.Builder(transaction)
+                        .withFilingId(dto.getReferenceAppointmentId())
+                        .withRequest(request)
+                        .build());
```

Comment:
Does this need to be logged at info level? My concern is the high number of logs from this service and whether this would end up being noise in the logs. I believe the transaction API logs the requests and responses.